tests/Unit/Domain/Creators/Test_AlignedLattice.cpp

```diff
@@ -44,16 +46,18 @@ void test_aligned_blocks(
 }  // namespace
 
 SPECTRE_TEST_CASE("Unit.Domain.Creators.AlignedLattice", "[Domain][Unit]") {
-  const auto domain_creator_1d =
-      TestHelpers::test_factory_creation<DomainCreator<1>>(
-          "AlignedLattice:\n"
-          "  BlockBounds: [[0.1, 2.6, 5.1, 5.2, 7.2]]\n"
-          "  IsPeriodicIn: [false]\n"
-          "  InitialGridPoints: [3]\n"
-          "  InitialLevels: [2]\n"
-          "  RefinedLevels: []\n"
-          "  RefinedGridPoints: []\n"
-          "  BlocksToExclude: []\n");
+  const auto domain_creator_1d = TestHelpers::test_factory_creation<
+      DomainCreator<1>, domain::OptionTags::DomainCreator<1>,
+      TestHelpers::domain::BoundaryConditions::
+          MetavariablesWithBoundaryConditions<1>>(
+      "AlignedLattice:\n"
+      "  BlockBounds: [[0.1, 2.6, 5.1, 5.2, 7.2]]\n"
+      "  IsPeriodicIn: [false]\n"
+      "  InitialGridPoints: [3]\n"
+      "  InitialLevels: [2]\n"
+      "  RefinedLevels: []\n"
+      "  RefinedGridPoints: []\n"
+      "  BlocksToExclude: []\n");
```

Comment:
This looks like it belongs in the next PR

Reply:
No, factory creation needs to be able to create all domain creators, and `Interval` no longer supports `void` as a metavariable